models/labels/addresses/infrastructure/identifier/cex_tokens/labels_cex_tokens.sql

```diff
@@ -1,65 +1,64 @@
-    , ('ethereum', '0x7be00ed6796b21656732e8f739fc1b8f1c53da0d', 'ACXT (ACDX Exchange)', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-15'), now(), 'cex_tokens', 'identifier')
-    , ('ethereum', '0x1b073382e63411e3bcffe90ac1b9a43fefa1ec6f', 'BEST (Bitpanda)', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-15'), now(), 'cex_tokens', 'identifier')
-    , ('ethereum', '0x50d1c9771902476076ecfc8b2a83ad6b9355a4c9', 'FTT (FTX)', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-15'), now(), 'cex_tokens', 'identifier')
-    , ('solana', 'AGFEad2et2ZJif9jaGpdMixQqvW5i81aBdvKe7PHNfz3', 'FTT (FTX)', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-15'), now(), 'cex_tokens', 'identifier')
-    , ('ethereum', '0xe7976c4efc60d9f4c200cc1bcef1a1e3b02c73e7', 'MAX (MAX Exchange)', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-15'), now(), 'cex_tokens', 'identifier')
-    , ('ethereum', '0x5b71bee9d961b1b848f8485eec8d8787f80217f5', 'BF (Bitforex)', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-15'), now(), 'cex_tokens', 'identifier')
-    , ('ethereum', '0x946551dd05c5abd7cc808927480225ce36d8c475', 'ONE (BigONE)', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-15'), now(), 'cex_tokens', 'identifier')
-    , ('bnb', '0x04baf95fd4c52fd09a56d840baee0ab8d7357bf0', 'ONE (BigONE)', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-15'), now(), 'cex_tokens', 'identifier')
-    , ('ethereum', '0xa2a54f1ec1f09316ef12c1770d32ed8f21b1fb6a', 'DFT (DigiFinex)', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-15'), now(), 'cex_tokens', 'identifier')
-    , ('ethereum', '0x7707aada3ce7722ac63b91727daf1999849f6835', 'BNK (Bankera)', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-15'), now(), 'cex_tokens', 'identifier')
-    , ('ethereum', '0x93b1e78a3e652cd2e71c4a767595b77282344932', 'BITO (BitoPro)', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-15'), now(), 'cex_tokens', 'identifier')
-    , ('bnb', '0xbfff3571f9fd637ae7cfb63ac2112fd18264ce62', 'TARM (Tarmex)', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-15'), now(), 'cex_tokens', 'identifier')
-    , ('ethereum', '0x1df7aa5551e801e280007dc0fc0454e2d06c1a89', 'BKK (BKEX)', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-15'), now(), 'cex_tokens', 'identifier')
-    , ('ethereum', '0xd4f6f9ae14399fd5eb8dfc7725f0094a1a7f5d80', 'BST (Bitsten)', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-15'), now(), 'cex_tokens', 'identifier')
-    , ('bnb', '0x1d89272821b3acc245acc1794e79a07d13c3e7e7', 'BST (Bitsten)', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-15'), now(), 'cex_tokens', 'identifier')
+    ('ethereum', 0xb8c77482e45f1f44de1745f52c74426c631bdd52, 'BNB (Binance)', 'infrastructure', 'hildobby', 'static', DATE '2023-03-15' , now(), 'cex_tokens', 'identifier')
```

Comment:
could use same timestamp approach from above?

keep in mind, these all union together downstream, so we need to be consistent across the spells in labels

Reply:
I will try to globally replace `DATE` to `from_iso8601_timestamp()` function in this PR.